src/api/sockets.js

```diff
@@ -19,35 +19,34 @@ function onConnection(socket) {
 
   socket.on('addToQueue', () => {
       if (!waitingRooms.includes(socket)) {
-      waitingRooms.push(socket)
+        waitingRooms.push(socket)
 
         if (waitingRooms.length >= 2) {
-          gameRoom = room += 1
+          console.log('second if')
+          room ++
```

Comment:
This may seem nit-picky, but it's really odd to see a space here: `room++` is the way I've always seen the increment operator used.